build.gradle

```diff
@@ -0,0 +1,274 @@
+                   '--exclude-components=nju,nic,llp,lg,gu,polynative,gvm,poly,nil,svml', '--force-bash-launchers=python,js,polybench', 'build'
+        }
+    }
+
+    doLast {
+        // Write current date to graal build dummy file. If this task is run again, the content of this file
+        // will change and subsequent tasks will run.
+        writeDateToFile(graalBuildDummy)
+    }
+}
+
+task getGraalBuildFolder(type: Exec) {
+    dependsOn buildGraal
+
+    inputs.files(graalBuildDummy)
+    standardOutput = new ByteArrayOutputStream()
+
+    commandLine 'ls', "$graalVMLatestDir"
+    doLast {
+        ext.graalVMDir = graalVMLatestDir + standardOutput.toString().replaceAll('\\r', '').replaceAll('\\n', '')
+    }
+}
+
+/**
+ * The command line util `tar` is being used rather Gradle's Tar task due to
+ * https://github.com/gradle/gradle/issues/3982, which describes symlinks being followed
+ * rather than preserved. Graal fails to run with the resolved symlinks.
+ */
+task tarGraal(type: Exec) {
+    dependsOn getGraalBuildFolder
+
+    // Task details
+    group = 'Other'
+    description = 'Generate graalvm.tar file.'
+
+    // Set the input to the folder in which the build is generated. We actually want to tar the only subdirectory from this.
+    inputs.dir(graalVMLatestDir)
+    outputs.file(graalVMTar)
+
+    doFirst {
+        if (environment.JVMCI_HOME.contains('openjdk1.8')) {
+            commandLine 'tar', 'czf', "$graalVMTar", '-C', getGraalBuildFolder.graalVMDir,
+                    "--exclude=./bin",
+                    "--exclude=./lib",
+                    "--exclude=./man",
+                    "--exclude=./sample",
+                    "--exclude=*src.zip",
+                    '.'
+        }
+        else {
+            commandLine 'tar', 'czf', "$graalVMTar", '-C', getGraalBuildFolder.graalVMDir,
+                    "--exclude=./lib/tools.jar",
+                    "--exclude=./lib/ct.sym",
+                    "--exclude=./lib/graalvm",
+                    "--exclude=./man",
+                    "--exclude=./sample",
+                    "--exclude=*src.zip",
+                    '.'
+        }
+    }
+}
+
+////////////////////////////////////////////////////////////////////////////////////////////////////
+// Maven-publish - Configuration for publishing maven artifacts
+////////////////////////////////////////////////////////////////////////////////////////////////////
+publishing {
+    // List of artifacts to publish
+    publications {
+
+        // Publish the tar file that contains graal
+        maven(MavenPublication) {
+
+            // Artifact to be publish
+            artifact(graalVMTar) {
+                extension 'tar.gz'
+                builtBy(tarGraal)
+            }
+
+            // Details about the artifact being publish
+            groupId = "com.r3.conclave.graal"
```

Comment:
Having just understood the requirements to publish to Maven Central, I think this different groupId to the normal Conclave one will cause delays. Can you revert this back to `com.r3.conclave`. See the blog post referenced in https://r3-cev.atlassian.net/browse/CON-1020 for more details.